spec/factories/users.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :user do
+    email "newuser@test.com"
+    password "password"
```

Comment:
unless you need to know what these are, you could also use `Faker` here (not required)